jsonpb/jsonpb_test_proto/test_objects.proto

```diff
@@ -33,6 +33,8 @@ syntax = "proto2";
 
 package jsonpb;
 
+import weak "gogoproto/gogo.proto";
```

Comment:
What is import weak?
Is there a way we could use this to not have the proto_path be so complicated i.e. not have the descriptor.proto and gogo.proto be in the proto_path, but still be able to use the gogoproto extensions?
Because that would be amazing.